### DISCUSSION TITLE
Example of training an image-to-image model from scratch with Diffusers

### DISCUSSION THREAD

**Discussion by sapkun** [huggingface/diffusers#7139] [General]
Hello, is there any sample code that shows how to build an image-to-image translation model from scratch (without fine-tuning an existing model) using Diffusers with the Stable Diffusion XL framework?

**Comment by sayakpaul** [comment]
Regular image-to-image translation is an extension of text-to-image (`StableDiffusionXLImg2ImgPipeline` for example). But for InstructPix2Pix, you can follow: https://github.com/huggingface/instruction-tuned-sd.
